en/components/grids_templates/editing.md

```diff
@@ -174,7 +175,7 @@ If you want to provide a custom template which will be applied when a cell is in
 > Any changes made to the cell's [`editValue`]({environment:angularApiUrl}/classes/igxcellcomponent.html#editvalue) in edit mode, will trigger the appropriate [editing event](#editing-events) on exit and apply to the [transaction state](batch-editing.md) (if transactions are enabled).
 
 > [!NOTE]
-> The cell editing template directive (`igxCellEditor`) is different from the [cell template(`igxCell`)](../grid/grid.md#cell-template) - the former handles how a column's cells **in edit mode** are displayed an controls the edited cell's **edit value**, the latter control how a column's cell's are show when **outside of edit mode**.
+> The cell editing template directive (`igxCellEditor`) is different from the [cell template(`igxCell`)](../grid/grid.md#cell-template) - the former handles how a column's cells **in edit mode** are displayed and controls the edited cell's **edit value**, the latter controls how a column's cells are show when **outside of edit mode**.
```

Comment:
Although not directly related to the PR, the hole Note can be updated to be a bit more "readable". Maybe:
>	The cell template igxCell controls how a column's cells are show when outside of edit mode.
>	The cell editing template directive igxCellEditor, handles how a column's cells in edit mode are displayed and controls the edited cell's edit value.